scripts/publish-github-release.sh

```diff
@@ -73,25 +73,25 @@ else
 fi
 
 # 4. Copy binaries from :sdk
```

Comment:
We can probably improve this step later.
I think we don't need to move this to that specific directory anymore, there is a copy task in VSTS.

Reply:
Yeah it actually works already without it, but this is nice to have when running locally